src/ucp/am/eager_single.c

```diff
@@ -458,6 +458,6 @@ ucp_proto_t ucp_am_eager_single_zcopy_reply_proto = {
     .init     = ucp_am_eager_single_zcopy_reply_proto_init,
     .query    = ucp_proto_single_query,
     .progress = {ucp_am_eager_single_zcopy_reply_proto_progress},
-    .abort    = ucp_proto_am_request_zcopy_abort,
+    .abort    = ucp_proto_request_zcopy_abort,
```

Comment:
can we really do this (i.e remove ucp_proto_am_request_zcopy_abort)?
what if ucp_am_eager_zcopy_pack_user_header was not called yet when we abort the operation?

Reply:
if it's indeed possible we can't